Gruntfile.js

```diff
@@ -9,7 +9,7 @@ module.exports = function (grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-contrib-uglify');
```

Comment:
run `grunt watch` and it will not compile and it will reload when you make
changes, just navigate to the '/build' directory instead of '/bin'

On Tue, Jul 19, 2016 at 6:43 PM, henktillman <notifications@github.com>
wrote:

> In Gruntfile.js
> <https://github.com/ripple/ripplecharts-frontend/pull/130#discussion_r71453859>
> :
>
> > @@ -9,7 +9,7 @@ module.exports = function (grunt) {
> >    grunt.loadNpmTasks('grunt-contrib-jshint');
> >    grunt.loadNpmTasks('grunt-contrib-concat');
> >    grunt.loadNpmTasks('grunt-contrib-watch');
> > -  grunt.loadNpmTasks('grunt-contrib-uglify');
>
> My mistake - it makes development much faster. When you run grunt to
> compile on your local machine, with uglify it's about 20 seconds, without
> about 3 seconds. I'll put it back in.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ripple/ripplecharts-frontend/pull/130/files/7c6aae70fc3fe6efee11a9e2d2ce2cd22a893772#r71453859>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD0McFIRs_BSBZrpbjZkxJaNPz8kG09Nks5qXX0xgaJpZM4JLAXU>
> .
>